.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+defaults: &defaults
```

Comment:
We can leave this file here for now while you're testing your PR. But in the end everything will need to run on Github Actions (using tox).

Reply:
Yes, that would be awesome. I will write a clear message somewhere here on the PR, when all the items are finished. I still might need two weeks.

Reply:
Ok, thanks.